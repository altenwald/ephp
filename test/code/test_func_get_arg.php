<?php
function foo($a, $b)
{
     $numargs = func_num_args();
     $b *= 10;
     echo "Number of arguments: $numargs\n";
     if ($numargs >= 2) {
         echo "Second argument is: " . func_get_arg(1) . "\n";
     }
}

foo(1, 2, 3);
