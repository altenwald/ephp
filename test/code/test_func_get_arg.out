Number of arguments: 3
Second argument is: 20
